src/test/fuzz/script.cpp

```diff
@@ -119,4 +125,41 @@ void test_one_input(const std::vector<uint8_t>& buffer)
             wit.SetNull();
         }
     }
+
+    const char* op_name = GetOpName(ConsumeOpcodeType(fuzzed_data_provider));
+    assert(op_name != nullptr);
+
+    const char* error_string = ScriptErrorString(static_cast<ScriptError>(fuzzed_data_provider.ConsumeIntegralInRange<int>(0, SCRIPT_ERR_ERROR_COUNT)));
+    assert(error_string != nullptr);
```

Comment:
unrelated nit: At compile time this can never be null. I'd prefer if the return value for strings was always `std::string`. Every caller of `ScriptErrorString` converts to `std::string` anyway.

Reply:
I agree, we should always return `std::string`. Will do that in a follow-up PR since that will touch validation code and thus need more scrutiny than adding tests.